app/controllers/repos_controller.rb

```diff
@@ -41,13 +41,26 @@ def create
     @repo   = Repo.search_by(params[:repo][:name], params[:repo][:user_name]).first unless params_blank?
     @repo ||= Repo.new(repo_params)
     if @repo.save
-      @repo_sub = current_user.repo_subscriptions.create(repo: @repo)
-      flash[:notice] = "Added #{@repo.to_param} for triaging"
-      RepoSubscription.background_send_triage_email(@repo_sub.id)
-      redirect_to @repo
+      @repo_sub = current_user.repo_subscriptions.find_or_initialize_by(repo: @repo)
+      
+      if @repo_sub.new_record?
+        @repo_sub.save
+        flash[:notice] = "Added #{@repo.to_param} for triaging"
+        RepoSubscription.background_send_triage_email(@repo_sub.id)
+      else
+        flash[:notice] = "Repository already exists"
+      end
+      redirect_to @repo      
     else
-      response = GitHubBub.get("/user/repos", type: "owner", token: current_user.token)
-      @own_repos = response.json_body
+      err = ""
+      @repo.errors.full_messages.each do |msg|
+        err += "<li> " + msg + "<br/>"
+      end
+
+      flash.now[:error] = err.html_safe
```

Comment:
@prathamesh-sonpatki this part of code is the ones that flashes the error which substitutes the one that I removed in app/views/repos/new.html.slim I did this so that the flash message will have the default bootstrap error UI